src/components/Confetti/Confetti.js

```diff
@@ -0,0 +1,12 @@
+const props = {
+  confettiParticles: {
+    type: Number,
+    default: 200,
```

Comment:
Please refactor this magic-number into a new `const defaultCountOfParticles = 200;` and then add a new `validator` function to this prop definition to prevent consumers of this component from using too-high values (add new `const maximumCountOfParticlesForSmoothPerformance = 400;` for example)